### DISCUSSION TITLE
SelectColumn trigger a Livewire event

### DISCUSSION THREAD

**Discussion by gotzsys** [filamentphp/filament#6934] [Help]
How do I trigger a Livewire event when changing select options?

```
SelectColumn::make('status')
    ->label('Status')
    ->options(self::FILTER_STATUS)
    ->toggleable()
    ->rules(['required']),
```

**Comment by pboivin** [comment]
The `SelectColumn` will automatically update and save the model. Not sure what you're trying to do but I think you're probably better using a `ViewColumn` with a custom `<select>` element? On there, you can use all the Livewire or Alpine attributes you want.
